frontend/src/APIClients/SimpleEntityAPIClient.ts

```diff
@@ -36,9 +36,16 @@ const create = async ({
     "accessToken",
   )}`;
   try {
+    // typescript {
     const { data } = await baseAPIClient.post("/simpleEntities", formData, {
       headers: { Authorization: bearerToken },
     });
+    // } typescript
+    // python {
```

Comment:
Do you mind changing the route name in the TypeScript backend as well so that we don't need to have 2 separate cases? I think kebab case URLs are much more commonly used than camelCase